### PR TITLE
Add the guide of 'Howto enable live boot'.

### PR DESCRIPTION
**Pull request by tanminger**
Signed-off-by: Ming Tan <ming.tan@intel.com>

### CODE REVIEW

**Comment by ttzeng** [comment]
@klynnrif, please also help to polish this article as well. Thanks in advance.

**Comment by ttzeng** [comment]
@klynnrif, please help to review this draft. Thanks!

File: source/tutorials/howto-enable-liveboot/howto-enable-liveboot.rst [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+   Note: please make sure all the buffered data are written to USB disk before plug out the USB disk, such as run 'sync' command in Linux.
+
+3. Boot from this USB disk to Android directly.
+
+   Plugin this USB disk, then power on, and press some function key to enter BIOS boot menu, select boot from this USB disk.
+
+   Then can boot to Android directly.
+
+   If you save some data to this Android, it will be saved to this USB disk, and you can try to use it in other computer.
```
Comment (klynnrif) [inline source/tutorials/howto-enable-liveboot/howto-enable-liveboot.rst]: Suggested rewrite: 
If you choose to save data to this Android device and you have not removed the USB disk, the data will be saved to the USB disk. You can then use the saved data on other computers.

File: source/tutorials/howto-enable-liveboot/howto-enable-liveboot.rst [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+
+Note:
+-----
+1. If you already install Android in the internal storage, please disable the internal storage if you want to try USB live boot.
```
Comment (klynnrif) [inline source/tutorials/howto-enable-liveboot/howto-enable-liveboot.rst]: Suggested rewrite: 
If you already installed Android in the internal storage, disable the internal storage if you are using USB live boot.

File: source/tutorials/howto-enable-liveboot/howto-enable-liveboot.rst [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+2. If you want to use one USB disk in two computers, then make sure the lock status are same in the two computers.
```
Comment (klynnrif) [inline source/tutorials/howto-enable-liveboot/howto-enable-liveboot.rst]: Suggested rewrite: 
If you want to use one USB disk in two computers, make sure the lock status is same in each computer.

**Pull request closed without merging**